.github/workflows/test_stdlib_releases_with_staging.yml

```diff
@@ -49,11 +49,11 @@ jobs:
           repository: 'ballerina-platform/ballerina-distribution'
           ref: ${{github.event.inputs.tag}}
 
-      - name: Set up JDK 11
-        uses: actions/setup-java@v2
+      - name: Set up JDK 17
```

Comment:
I think we have to update the `actions/checkout@v3` as well.

Reply:
Ack

Reply:
Updated the versions